Add identity and access control docs

jraddaoui opened the pull request:
Create an administrator section in the docs and add an initial identity and access control section.

Refs #957.

fiver-watson commented on docs/src/admin-manual/iac.md:
```diff
@@ -0,0 +1,152 @@
+They must match the ones configured in the API. `VITE_OIDC_AUTHORITY` has to be
+the same OIDC provider URL and `VITE_OIDC_CLIENT_ID` needs to be the same or a
+trusted client. This client (and the one used in the API configuration, if they
+are not the same) must be included in the `aud` claim from the access token,
```
ends w a comment instead of a period

fiver-watson commented on docs/src/admin-manual/iac.md:
```diff
@@ -0,0 +1,152 @@
+scopes. The ABAC variables will work in the same way as they do in the API,
+they are explained in detail in the API configuration comments above.
+
+These environment variables must be set at build time or they can be replaced
```
you say "must", and then follow with an "or" - is the or a bad outcome if you don't do the _must_? Or is it an alternative? If it's an alternative, then I would change the "must" to "can" - _must_ is typically used as an imperative (you must do this, or else...) 

> These environment variables can be set at build time, or they can be replaced in the final assets.

fiver-watson reviewed (approved):
Overall, looks good - 2 minor grammatical suggestions

jraddaoui committed:
Add identity and access control docs

Create an administrator section in the docs and add an initial identity
and access control section.

jraddaoui commented:
Thanks @fiver-watson!

@djjuhasz @sbreker, I'll merge so it gets deployed to the docs site. I'll still appreciate your feedback when you have some time, will address it in a follow-up PR.
